src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.Unix.cs

```diff
@@ -335,20 +335,19 @@ internal void LogBuffer(int size)
         private SocketError FinishOperationAccept(Internals.SocketAddress remoteSocketAddress)
         {
             System.Buffer.BlockCopy(_acceptBuffer!, 0, remoteSocketAddress.Buffer, 0, _acceptAddressBufferCount);
-            Socket? sukru = _acceptSocket;
+            Socket? tempAcceptSocket = _acceptSocket;
```

Comment:
Instead of tempAcceptSocket, should this be callerSuppliedAcceptSocket, or something like that?

Reply:
Yeah, your proposal is more appropriate.